documentation/book/proc-manual-restart-kafka.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// assembly-todo.adoc
+
+[id='proc-manual-restart-kafka-{context}']
+= Manually restart a Kafka cluster
+
+This procedure describes how to manually restart an existing Kafka cluster by using an {ProductPlatformName} annotation.
+
+.Prerequisites
+
+* A running Kafka cluster.
+* A running Cluster Operator.
+
+.Procedure
+
+. Find the name of `StatefulSet` which controls Kafka pods to restart.
++
+Lets say the pod is named _my-cluster-kafka-x_. The name of the StatefulSet is _my-cluster-kafka_.
```

Comment:
I am not quite sure that we need this. We are talking about "rolling update" or "restart" an entire cluster. It means that user should know the name of the SS to annotate starting from the name of the cluster but with no relation with how the pods are named. Btw just my 2 cents, @scholzj ?

Reply:
I think starting from a cluster name which would be known to the user is fine.